provider/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '0.5.0'
+__version__ = '0.6.0'
```

Comment:
Since this is a backward incompatible change, should we update the major version to 1?